src/core/lib/iomgr/error.h

```diff
@@ -0,0 +1,143 @@
+/*
+ *
+ * Copyright 2015, Google Inc.
```

Comment:
I suspect it perhaps doesn't work for newly added files, just for modified ones... to be fixed regardless.